aspnetcore/signalr/version-differences.md

```diff
@@ -19,6 +19,7 @@ ASP.NET Core SignalR isn't compatible with clients or servers for ASP.NET Signal
 | Server NuGet Package | [Microsoft.AspNet.SignalR](https://www.nuget.org/packages/Microsoft.AspNet.SignalR/) | [Microsoft.AspNetCore.App](https://www.nuget.org/packages/Microsoft.AspNetCore.App/) (.NET Core)<br>[Microsoft.AspNetCore.SignalR](https://www.nuget.org/packages/Microsoft.AspNetCore.SignalR/) (.NET Framework) |
 | Client NuGet Packages | [Microsoft.AspNet.SignalR.Client](https://www.nuget.org/packages/Microsoft.AspNet.SignalR.Client/)<br>[Microsoft.AspNet.SignalR.JS](https://www.nuget.org/packages/Microsoft.AspNet.SignalR.JS/) | [Microsoft.AspNetCore.SignalR.Client](https://www.nuget.org/packages/Microsoft.AspNetCore.SignalR.Client/) |
 | Client npm Package | [signalr](https://www.npmjs.com/package/signalr) | [@aspnet/signalr](https://www.npmjs.com/package/@aspnet/signalr) |
+| Java Maven Package |  | [com.microsoft.signalr](https://search.maven.org/artifact/com.microsoft.signalr/signalr) |
```

Comment:
Hmmm, I don't think we should leave this blank since there technically was a client. Maybe note that the client it deprecated/obsolete/Archived?

Reply:
@bradygaster I agree with Mikael. It may be confusing to leave this blank. Thoughts?

Reply:
Fixing this in the next few hours, apologies for the delay.